examples/Adsorption/Generator/adsorption_autopas_tuning.xml

```diff
@@ -87,7 +87,7 @@
         <allowedTraversals>c01, c04, c08, c18, sli</allowedTraversals>
         <allowedContainers>linkedCells</allowedContainers>
         <selectorStrategy>fastestMedian</selectorStrategy>
-        <tuningStrategy>FullSearch</tuningStrategy><!--FullSearch, BayesianSearch, ...-->
+        <tuningStrategies>FullSearch</tuningStrategies><!--FullSearch, BayesianSearch, ...-->
```

Comment:
In `examples/Argon/200K_18mol_1/config_autopas_allOptions.xml`, you write to leave empty for FullSearch (similar to md-flexible). Is this not the case here as well?

Reply:
correct, should be removed. -> 4a1f9c7a8